widgetssdk/src/main/java/com/glia/widgets/config/UiComponents.java

```diff
@@ -0,0 +1,75 @@
+package com.glia.widgets.config;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.graphics.drawable.Drawable;
+import android.util.TypedValue;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.glia.widgets.config.components.RemoteColor;
+import com.glia.widgets.config.components.RemoteConfiguration;
+import com.glia.widgets.config.components.base.Alignment;
+import com.glia.widgets.config.components.base.DpSize;
+import com.glia.widgets.config.components.base.SpSize;
+import com.glia.widgets.config.components.base.TextStyle;
+import com.glia.widgets.config.deserializers.AlignmentDeserializer;
+import com.glia.widgets.config.deserializers.DpDeserializer;
+import com.glia.widgets.config.deserializers.RemoteColorDeserializer;
+import com.glia.widgets.config.deserializers.SpDeserializer;
+import com.glia.widgets.config.deserializers.TextStyleDeserializer;
+import com.google.android.material.shape.CornerFamily;
+import com.google.android.material.shape.MaterialShapeDrawable;
+import com.google.android.material.shape.ShapeAppearanceModel;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+public class UiComponents {
+    private static float applyDimension(int unit, float value, Context context) {
+        return TypedValue.applyDimension(unit, value, context.getResources().getDisplayMetrics());
+    }
+
+    public static float dpToPx(float value, Context context) {
+        return applyDimension(TypedValue.COMPLEX_UNIT_DIP, value, context);
+    }
+
+    public static float spToPx(float value, Context context) {
+        return applyDimension(TypedValue.COMPLEX_UNIT_SP, value, context);
+    }
+
+    /**
+     * @return {@link Gson} instance with applied deserializers to parse remote config.
+     */
+    public static Gson getGsonInstance() {
+        return new GsonBuilder()
+                .registerTypeAdapter(RemoteColor.class, new RemoteColorDeserializer())
+                .registerTypeAdapter(DpSize.class, new DpDeserializer())
+                .registerTypeAdapter(SpSize.class, new SpDeserializer())
+                .registerTypeAdapter(TextStyle.class, new TextStyleDeserializer())
+                .registerTypeAdapter(Alignment.class, new AlignmentDeserializer())
+                .create();
+    }
+
+    @Nullable
+    public static RemoteConfiguration parseRemoteConfiguration(@NonNull String remoteConfiguration) {
+        return getGsonInstance().fromJson(remoteConfiguration, RemoteConfiguration.class);
+    }
+
+    /**
+     * [WIP] example how to create drawable with rounded corners and stroke.
```

Comment:
NIT: I get it that this is WIP. But to have the track of things and since it is not mentioned in TODO I wanted to mention that the numbers could be also extracted and/or commented

Reply:
Actually, this needs to be removed. I created this as a draft, but it seems no need for it anymore.